configs/11.0/deb/monolithic/devel.postinst

```diff
@@ -32,6 +32,6 @@ if [[ "${1}" == configure ]]; then
 		mkdir -p /usr/share/modules/modulefiles
 	fi
 	if [[ -w /usr/share/modules/modulefiles/. ]]; then
-		ln -s __AT_DEST__/share/modules/modulefiles/__AT_VER_REV_INTERNAL__ /usr/share/modules/modulefiles/.
+		ln -s __AT_DEST__/share/modules/modulefiles/__AT_MAJOR_INTERNAL__ /usr/share/modules/modulefiles/.
```

Comment:
Which directory will be created for AT 11.0-3 rc1?  Would it conflict with AT 11.0-2 rc1?

Reply:
The motivation for this change came from observing that the build with which I was working was creating an environment modulefile named `at11.0-1`.  The `-1` part is an annoyance for the user, and insignificant as the environment modulefile should be common for a release (replaced upon update from AT 11.0-1 to AT 11.0-2, for example).

`__AT_VER_REL_INTERNAL__` included the full "_version.release-revision_" string, and is undesirable.

`__AT_MAJOR_INTERNAL__` for external releases will just be "_version.release_", as desired.
`__AT_MAJOR_INTERNAL__` for internal releases, like "_alpha_", "_beta_", "_rc1_", will include the extra `INTERNAL` string appended, like `at11.0-alpha`, etc., so will not conflict when installed side-by-side with other internal releases.

Reply:
Relevant lines from `Makefile`:
```
AT_VER_REV := $(AT_MAJOR_VERSION)-$(AT_REVISION_NUMBER)
AT_MAJOR := $(AT_NAME)$(AT_MAJOR_VERSION)
ifeq ($(AT_INTERNAL),none)
    AT_FULL_VER := $(AT_VER_REV)
else
    AT_FULL_VER := $(AT_VER_REV)-$(AT_INTERNAL)
endif
AT_VER_REV_INTERNAL := $(AT_NAME)$(AT_FULL_VER)
```
```
ifneq ($(AT_INTERNAL),none)
    AT_MAJOR_INTERNAL := $(AT_MAJOR)-$(AT_INTERNAL)
    AT_DIR_NAME ?= $(AT_VER_REV_INTERNAL)
else
    AT_MAJOR_INTERNAL := $(AT_MAJOR)
    AT_DIR_NAME ?= $(AT_MAJOR)
endif
```

Reply:
> __AT_MAJOR_INTERNAL__ for internal releases, like "alpha", "beta", "rc1", will include the extra INTERNAL string appended, like at11.0-alpha, etc., so will not conflict when installed side-by-side with other internal releases.

@ThinkOpenly , I think you meant to say they **will conflict**.  Both AT 11.0-1 rc1 and AT 11.0-2 rc1 will store in the same directory: `at11.0-rc1`

Reply:
I did mean **not conflict**, but perhaps I don't understand.
For clarity:, where will the following install?
- AT 11.0-1 rc1: ` /opt/at11.0-1-rc1`?
- AT 11.0-2 rc2:  `/opt/at11.0-1-rc2`?
- AT 11.0-1: `/opt/at11.0`?
(If so, note the inconsistency, as the latter has completely dropped the revision. How does that magically happen?)
Most importantly, I want the released version to be called `at11.0` (not `at11.0-1`), because all of the released versions of the revisions will overwrite, correct?

Reply:
> For clarity:, where will the following install?
>- AT 11.0-1 rc1: ` /opt/at11.0-1-rc1`?
>- AT 11.0-2 rc2:  `/opt/at11.0-1-rc2`?
>- AT 11.0-1: `/opt/at11.0`?

Yes, and this happens due to the lines of code you quoted in your last comment:
````
ifneq ($(AT_INTERNAL),none)
    AT_MAJOR_INTERNAL := $(AT_MAJOR)-$(AT_INTERNAL)
    AT_DIR_NAME ?= $(AT_VER_REV_INTERNAL)
else
    AT_MAJOR_INTERNAL := $(AT_MAJOR)
    AT_DIR_NAME ?= $(AT_MAJOR)
endif
````

When we have an internal release the directory name is set to AT_VER_REV_INTERNAL.